service/controller/v11/resource/deployment/master_deployment.go

```diff
@@ -89,21 +89,20 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 				Replicas: &replicas,
 				Template: apiv1.PodTemplateSpec{
 					ObjectMeta: apismetav1.ObjectMeta{
-						GenerateName: key.MasterID,
-						Finalizers: []string{
-							key.DrainingNodesFinalizer,
+						Annotations: map[string]string{
+							key.AnnotationIp:          "",
+							key.AnnotationService:     key.MasterID,
+							key.APIEndpointAnnotation: key.ClusterAPIEndpoint(customObject),
```

Comment:
Nitpick: would be nice to name it `AnnotaionAPIEndpoint` as the rest.

Reply:
Done.